### ISSUE TITLE
Unable to add a G Suite for Education account

### ISSUE THREAD

**Issue by thatgeek95** [Thomas101/wmail#477]
* Which version of WMail are you using? [e.g. 1.3.1]
2.0.0

* Which Operating System are you using? [e.g. Windows 10, Mac OSX 10, Ubuntu 14.04]
Ubuntu 16.04

When I try to add my G Suite for Education account which uses my University's SSO service to authenticate, I am never taken to the SSO screen, I just see a white screen after the initial email address entry screen.

**Comment by albert1t0** [comment]
I have the same problem ... blank screen for login, try alberto.torreblanca@pucp.edu.pe 
Version 2.0.7
OS Ubuntu 16.04

**Comment by Thomas101** [comment]
Try going into settings and then the advanced tab. There's an option in there called "Ignore GPU blocklist". Toggle it so it's on and restart wmail and try again. I think this might be an Ubuntu issue

**Comment by thatgeek95** [comment]
Turned that on and it doesn't work unfortunately, same thing happening.

**Comment by Thomas101** [comment]
Do you know if you're using a proxy server? If you are you may need to configure this under `settings > advanced` before signing in

**Comment by thatgeek95** [comment]
I do not use a proxy server.

**Comment by thatgeek95** [comment]
No errors appear

**Comment by Thomas101** [comment]
@albert1t0 I've just tried your email address and I get forwarded through to here...

<img width="200" alt="screen shot 2017-01-08 at 14 48 33" src="https://cloud.githubusercontent.com/assets/103586/21750720/9bdaae3e-d5b1-11e6-8dce-c50249a2b8b7.png">

@thatgeek95 do you mind me asking what domain your email address is for? For example if it's `myusername@myemailaddress.com` it would be `myemailaddress.com`.

If either of you try adding a standard google account does that work successfully?

**Comment by thatgeek95** [comment]
Domain is @scarletmail.rutgers.edu.

**Comment by thatgeek95** [comment]
Also, yes I can add a standard Google account.

Thank you for the help!

**Comment by albert1t0** [comment]
Hi @Thomas101, with the new version 2.0.9 all work great. Thanks!!!

**Comment by narayanan** [comment]
I'm using version 2.0.9, have enabled "Ignore GPU blocklist" option in the Advanced Settings, restarted the app. Still seeing a blank screen after entering the email address. No error / log in console. So not sure whats going on. Let me know if I can provide additional information to investigate the issue further.

**Comment by narayanan** [comment]
I managed to sign-in to my corporate account. Here is more details that might help

- When google re-directs to the corporate sign-on, the sign-on page automatically redirects me back to google without challenging for user name / password. This happens when my laptop is in corporate network or if I'm already signed in. I'm seeing a blank screen in this case. 
- When I disconnected from the corporate network, username / password dialog appears and am able to successfully sign on & add the account.

Basically I think blank screen appears when the single sign on authenticates user with out challenging for credentials.

**Comment by Thomas101** [comment]
Ah this is interesting. Just to double check the flow in the first case...
1.) You Click add account
2.) Enter your email address on the google page
3.) Then the corporate sign-on loads, checks your on the corporate network, authenticates you and redirects back to google - which should leave you in a signed in state, but wmail doesn't pick this up

**Comment by narayanan** [comment]
@Thomas101 Yes, the flow you explained is correct.

**Comment by Thomas101** [comment]
Okay, thanks I'll see if I can run a third party auth server my end to see if I can get it to fail in a similar way

**Comment by Thomas101** [comment]
Out of curiosity, have you been able/or used any third party services in your web browser that authenticate with google. For example signing in to wunderlist.com with your google account? Have you had any trouble with these?

**Comment by narayanan** [comment]
I did not try with other third party services. As mine is a corporate account, there are restrictions on what services I can use.

**Comment by thatgeek95** [comment]
Updated to the latest release version (2.1.0) and the issue is resolved. Thank you!!
